client/src/main/scala/drt/client/services/handlers/StaffAssignmentsHandler.scala

```diff
@@ -0,0 +1,57 @@
+package drt.client.services.handlers
+
+import diode.Implicits.runAfterImpl
+import diode._
+import diode.data.{Pot, Ready}
+import drt.client.actions.Actions._
+import drt.client.logger.log
+import drt.client.services.JSDateConversions.SDate
+import drt.client.services.{DrtApi, PollDelay, ViewMode}
+import drt.shared.ShiftAssignments
+import upickle.default.{read, write}
+
+import scala.concurrent.Future
+import scala.concurrent.duration._
+import scala.language.postfixOps
+import scala.scalajs.concurrent.JSExecutionContext.Implicits.queue
+
+class StaffAssignmentsHandler[M](getCurrentViewMode: () => ViewMode, modelRW: ModelRW[M, Pot[ShiftAssignments]]) extends LoggingActionHandler(modelRW) {
+  def scheduledRequest(viewMode: ViewMode): Effect = Effect(Future(GetShifts(viewMode))).after(2 seconds)
+
+  protected def handle: PartialFunction[Any, ActionResult[M]] = {
+    case SetStaffShifts(viewMode, shifts, _) =>
```

Comment:
Shall we rename these action case classes to match their actions? eg `SetStaffAssignments`, `GetStaffAssignments` etc